Dockerfile

```diff
@@ -30,12 +35,27 @@ RUN ln --symbolic /app/privaterelay/locales/pt-BR/ privaterelay/locales/pt
 RUN ln --symbolic /app/privaterelay/locales/es-ES/ privaterelay/locales/es
 COPY --chown=app .env-dist /app/.env
 
+# TODO The email tracker list commands are a duplicate of a CircleCI job
+# https://github.com/jbuck/fx-private-relay/blob/57cdfc5421b5faf0fe1f228aeb524d4232a221e0/.circleci/python_job.bash#L69-L77
+RUN python manage.py get_latest_email_tracker_lists --skip-checks
+RUN python manage.py get_latest_email_tracker_lists --skip-checks --tracker-level=2
+
 # Collect all staticfiles, including for apps that may be disabled
+COPY --from=frontend --chown=app /home/node/frontend/out frontend/out
 RUN PHONES_ENABLED=True \
     API_DOCS_ENABLED=True \
     mkdir -p /app/staticfiles && \
     python manage.py collectstatic --no-input -v 2
 
+# These arguments change frequently so define them last
```

Comment:
praise: nice!

Reply:
Can't take credit for this! It's @jbuck's work, just moved to the `Dockerfile`